frontend/packages/ceph-storage-plugin/src/constants/kms.ts

```diff
@@ -3,6 +3,7 @@ import { VaultConfig, HpcsConfig, ProviderNames, KmsImplementations, KMSConfig }
 export const KMSMaxFileUploadSize = 4000000;
 export const KMSConfigMapName = 'ocs-kms-connection-details';
 export const KMSConfigMapCSIName = 'csi-kms-connection-details';
+export const KMSVaultTokenSecretName = 'ocs-kms-token';
 export const KMSSecretName = 'ocs-kms-token';
```

Comment:
Why same values in two different constants?

Reply:
ack, i renamed KMSSecretName to KMSVaultTokenSecretName but forgot to remove old one